packages/subgraph/schema.graphql

```diff
@@ -1,13 +1,256 @@
-type Token @entity {
+type TenderizeGlobal @entity { # Global Values/ TODO: better name?
+  "Fixed ID of '1'"
   id: ID!
-  decimals: Int
-  name: String
-  symbol: String
+  "List of congifs for all integrations"
+  configs: [Protocol!]!
+  # TODO: Add global totals 
 }
 
-type Transfer @entity {
+type Protocol @entity {
+  "Name of the protocol eg. 'livepeer'"
   id: ID!
+  "Tenderizer Data"
+  tenderizer: Tenderizer @derivedFrom(field: "protocol")
+  "TenderFarm Data"
+  tenderFarm: TenderFarm @derivedFrom(field: "protocol")
```

Comment:
Since there is a one-to-one relation ship between the `Protocol` entity and a `Tenderizer` and `TenderFarm` there is no use for derived fields here

Reply:
Yea, I mainly added cuz it would auto populate, but now that I think of it, it's not worth adding the reverse lookup and derived field for a one-to-one relationship, better done manually. We can lose the reverse lookups below as well in that case.